.github/workflows/commit-queue.yml

```diff
@@ -37,6 +37,7 @@ jobs:
                   --repo ${{ github.repository }} \
                   --base ${{ github.ref_name }} \
                   --label 'commit-queue' \
+                  --no-label 'blocked' \
```

Comment:
What about also `wip`?

Reply:
I'll do in a followup

Reply:
Maybe we should remove the `wip` label, it was created at a time where draft PRs were not a thing